Adjust some default settings (default shipped .conf files) for 
openSUSE and openSUSE based appliances.

Removing default LiveCD/LiveUSB user is not mandatory, 
but "linux" user is no longer needed after installation.
Note, that if user not exist, then Camamares crash!
See https://github.com/calamares/calamares/issues/577
Thus here we justion mention this option, but not enabling.

diff --git a/settings.conf b/settings.conf
--- a/settings.conf
+++ b/settings.conf
@@ -85,16 +82,18 @@
 #  - dracutlukscfg
 #  - plymouthcfg
-  - initcpiocfg
-  - initcpio
+#  - initcpiocfg
+#  - initcpio
+# - removeuser
   - users
   - displaymanager
   - networkcfg
   - hwclock
-  - services
-#  - dracut
-  - initramfs
-#  - grubcfg
+#  - services
+  - dracut
+#  - initramfs
+  - grubcfg
   - bootloader
+  - packages
   - umount
 - show:
 #  - webview@owncloud
@@ -116,7 +112,7 @@
 # is shown.
 # Default is false.
 # YAML: boolean.
-prompt-install: false
+prompt-install: true
 
 # If this is set to true, Calamares will execute all target environment commands in the
 # current environment, without chroot. This setting is considered experimental, and it
